server/controllers/UsersController.js

```diff
@@ -1,9 +1,79 @@
-import models from '../models/Users';
+import { Client } from 'pg';
+import bcrypt from 'bcryptjs';
+import jwt from 'jsonwebtoken';
+import database from '../models/database';
 
-class UsersController {
-  static createUsers(req, res) {
-    models.userSignUp(req, res);
+class Users {
+  static userSignUp(req, res) {
+    const connectString = database;
+    const clientString = new Client(connectString);
```

Comment:
What is a client connection instantiation doing in your Controller? This should be abstracted